sql/core/src/main/scala/org/apache/spark/sql/DataFrame.scala

```diff
@@ -448,6 +461,10 @@ class DataFrame private[sql](
    * @since 1.3.0
    */
   def join(right: DataFrame, joinExprs: Column, joinType: String): DataFrame = {
+    join(right, Seq(joinExprs), joinType)
+  }
+
+  def join(right: DataFrame, joinExprs: Seq[Column], joinType: String): DataFrame = {
```

Comment:
So for `def join(self, other, joinExprs=None, joinType=None)` we don't support given a list of `Column` as `joinExprs`?